deploy/02_Deploy_APIConsumer.js

```diff
@@ -21,22 +21,19 @@ module.exports = async ({
     linkTokenAddress = networkConfig[chainId]['linkToken']
     oracle = networkConfig[chainId]['oracle']
   }
-  let jobId = networkConfig[chainId]['jobId']
-  let fee = networkConfig[chainId]['fee']
+  const jobId = networkConfig[chainId]['jobId']
+  const fee = networkConfig[chainId]['fee']
 
-  log("----------------------------------------------------")
-  log('Deploying APIConsumer')
```

Comment:
we don't like these?

Reply:
They're already logged with `log: true` in `deploy`.